leader-election-impl/src/main/java/com/palantir/paxos/PaxosAcceptorImpl.java

```diff
@@ -118,11 +121,20 @@ public BooleanPaxosResponse accept(long seq, PaxosProposal proposal) {
 
     @Override
     public long getLatestSequencePreparedOrAccepted() {
+        Stopwatch timer = Stopwatch.createStarted();
+        Thread currentThread = Thread.currentThread();
+        logger.debug("[{} - {}] Entering getLatestSequencePreparedOrAccepted", currentThread.getName(), currentThread.getId());
+
+        long result;
         if (state.isEmpty()) {
-            return greatestInLogAtStartup;
+            result = greatestInLogAtStartup;
         } else {
-            return Math.max(greatestInLogAtStartup, state.lastKey());
+            result = Math.max(greatestInLogAtStartup, state.lastKey());
```

Comment:
I don't imagine this one will be slow at all (local ops), but to be fair it is still useful for evaluating hypotheses.

Reply:
Yep agree. This is just a "make sure I'm not crazy" check really.

Reply:
I think that if the remoting layer is the issue (and it probably is) we'll see that by noting that the start timestamp of the timing call added here is much later than the time the request arrives at the server (by taking packet capture).